core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -228,6 +239,26 @@ class ConfigCommandTest extends ZooKeeperTestHarness with Logging {
     verifyAlterBrokerConfig(node, "1", List("--entity-name", "1"))
   }
 
+  @Test
+  def shouldAddBrokerLoggerConfig(): Unit = {
```

Comment:
Did you consider also testing describe/alter on all brokers? (i.e. when entity name is empty)?
It is supported for broker configs, so either it works for logs too, or we need to document (and have an explicit error) that it doesn't.

Reply:
We do have an explicit error:
```scala
if (entityTypeVals.contains(EntityType.User) || entityTypeVals.contains(EntityType.Client) || entityTypeVals.contains(EntityType.Broker)) {
          if (!options.has(entityName) && !options.has(entityDefault))
            throw new IllegalArgumentException("--entity-name or --entity-default must be specified with --alter of users, clients or brokers")
        } else if (!options.has(entityName))
            throw new IllegalArgumentException(s"--entity-name must be specified with --alter of $entityTypeVals")
```
the `throw new IllegalArgumentException(s"--entity-name must be specified with --alter of $entityTypeVals")` should catch this.
great call on having a test case - adding one